api/v1alpha2/linodecluster_types.go

```diff
@@ -98,11 +98,21 @@ type LinodeCluster struct {
 	Status LinodeClusterStatus `json:"status,omitempty"`
 }
 
-func (lm *LinodeCluster) GetConditions() clusterv1.Conditions {
+func (lm *LinodeCluster) GetConditions() []metav1.Condition {
```

Comment:
Should we use `lc` for LinodeCluster instead of `lm`? Its just a variable name, but I think lc makes it more easier to refer to LinodeCluster